packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -108,6 +108,13 @@ import {
   S3BucketCard,
   LambdaFunctionCard,
   IAMUserCard,
+  IAMRoleCard,
+  EKSClusterCard,
+  isEksClusterAvailable,
```

Comment:
The following 5 functions are to enable the entity swticher. Basically they are boolean functions, returning true if the entity has the right annotation. This one returns true only if the entity has "amazon.com/aws-cluster-arn"